core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -220,6 +220,10 @@ import akka.stream.scaladsl.Source
         else result.map(_ => Done)
       result2.pipeTo(sender())
 
+    case DeleteEventsTo(persistenceId, toSequenceNr) =>
```

Comment:
what's the difference of this and the existing `akka.persistence.DeleteMessagesTo`?

Reply:
That one doesn't respond to the sender() but the persistentActor in the message

Reply:
ok, but you could use the other ask to pass in the "ask sender" in the message. You find it in ExplicitAskSupport